src/file-4.xoo

```diff
@@ -0,0 +1,3 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:36123/organizations/org0/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:36123/project/issues?id=sample&branch=feature3&issues=AWFDtmi5kWY7G52pgBq4&open=AWFDtmi5kWY7G52pgBq4)

[ISSUE_KEY]: AWFDtmi5kWY7G52pgBq4